src/mqt/bench/tket_helper.py

```diff
@@ -1,23 +1,29 @@
 from __future__ import annotations
 
 from pathlib import Path
+from typing import TYPE_CHECKING
 
 from mqt.bench import utils
-from pytket import OpType, architecture, circuit
+from pytket import OpType
+from pytket.architecture import Architecture  # type: ignore[attr-defined]
 from pytket.extensions.qiskit import qiskit_to_tk
-from pytket.passes import (
+from pytket.passes import (  # type: ignore[attr-defined]
```

Comment:
Otherwise I get `error: Module "pytket.passes" has no attribute <xyz>`. Would there be any other way to deal with this?

Reply:
Just checked. We do the same thing in the predictor package. And now I also remember the reason. That's because these passes come from compiled extensions in TKET and they do not provide type stubs for those yet. Feel free to resolve this.